src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/ValueTask.cs

```diff
@@ -56,8 +56,8 @@ namespace System.Threading.Tasks
     [StructLayout(LayoutKind.Auto)]
     public readonly struct ValueTask : IEquatable<ValueTask>
     {
-        /// <summary>A task canceled using `new CancellationToken(true)`.</summary>
-        private static readonly Task s_canceledTask = Task.FromCanceled(new CancellationToken(canceled: true));
+        /// <summary>A task canceled using `new CancellationToken(true)`. Lazily created only when first needed.</summary>
+        private static volatile Task? s_canceledTask;
```

Comment:
I would expect it to be initialized lazily already given that it is the only static initialization in the type. Is there anything else statically initialized in `ValueTask` ? 

I think this is still a good change (minor code size improvement, future proofing - in case there is some other static initialization added in future).

Reply:
> Is there anything else statically initialized in ValueTask ?

Not currently.  I hit this as part of trying out some refactoring where I inadvertently added some more, and then saw this manifesting in a trace.  So it's the "in case there is some other static initialization added in the future" case you called out.  Figured I'd clean it up now while it was top of mind.